hips/utils/testing.py

```diff
@@ -43,19 +42,19 @@ def make_test_wcs_geometry(case=0):
     if case == 0:
         return WCSGeometry.create(
             skydir=SkyCoord(3, 4, unit='deg', frame='galactic'),
-            shape=(2, 3), coordsys='GAL',
+            height=2, width=3, coordsys='galactic',
```

Comment:
I'd suggest we always write in the order `width`, then `height`, even if it doesn't matter when calling with kwargs.  Please change here and below.